src/pages/login/index.js

```diff
@@ -26,31 +28,31 @@ const Login = () => {
   return (
     <div className="bg-blue login credentialpage">
       <CredentialsCard
-        title="Login"
-        socialLinksTitle="Or log in with"
-        altButtonTitle="Need an account?"
+        title={t('login')}
+        socialLinksTitle={t("loginAlternative")}
+        altButtonTitle={t("needOtherAccount")}
         altButtonLink="/register"
-        altButtonText="Sign up"
+        altButtonText={t("Registrera dig")}
       >
         <div className="login-form">
           <form>
             <TextInput
               value={formData.email}
               onChange={onChange}
               name="email"
-              label={"Email *"}
+              label={`${t('email')} *`}
```

Comment:
I don't think you need to use backticks here. Because the result of calling "t" is returning a straight string. Did this still work in testing?